app/src/models/repository.ts

```diff
@@ -112,3 +112,24 @@ export function nameOf(repository: Repository) {
 
   return gitHubRepository !== null ? gitHubRepository.fullName : repository.name
 }
+
+/**
+ * Get the GitHub html URL for a repository, if it has one.
+ * Will return the parent GitHub repository's URL if it has one.
+ * Otherwise, returns null.
+ */
+export function getGitHubHtmlUrl(repository: Repository): string | null {
+  if (repository.gitHubRepository === null) {
+    return null
+  }
+  if (
+    repository.gitHubRepository.parent !== null &&
+    repository.gitHubRepository.parent.htmlURL !== null
+  ) {
+    return repository.gitHubRepository.parent.htmlURL
+  }
+  if (repository.gitHubRepository.htmlURL) {
```

Comment:
Should we add `!== null` here to be consistent with the other checks?

```suggestion
  if (repository.gitHubRepository.htmlURL !== null) {
```

Reply:
yes!